### PR TITLE
pop up box for Kab CR apply

### PR DESCRIPTION
**Pull request by davco01a**
pop up box for Kab CR apply

### CODE REVIEW

File: scripts/install-kabanero-foundation.sh [hunk 1]
```diff
@@ -138,8 +138,13 @@ else
...
-echo "instance:"
-echo "oc apply -n ${namespace} -f ${SAMPLE_KAB_INSTANCE}"
+echo "***************************************************************************"
+echo "*                                                                         *"
+echo "*  The installation script is complete.  You can now create an instance   *"
+echo "*  of the Kabanero CR.  If you have cloned and curated a collection set,  *"
+echo "*  apply the Kabanero CR that you created.  Or, to create the default     *"
+echo "*  instance:                                                              *"
+echo "*                                                                         *"
+echo "*      oc apply -n ${namespace} -f ${SAMPLE_KAB_INSTANCE}                 *"
```
Comment (kaczyns) [inline scripts/install-kabanero-foundation.sh]: I am all for drawing more attention to this area, but you've drawn a very pretty box here that is going to look funny when `${namespace}` and `${SAMPLE_KAB_INSTANCE}` are expanded.  Would it be better to leave off the right edge of the box?

Comment (davco01a) [reply scripts/install-kabanero-foundation.sh]: sure

Comment (davco01a) [reply scripts/install-kabanero-foundation.sh]: done

**Comment by kaczyns** [comment]
Thanks for the PR Dave - I'm merging

**Pull request merged**